easybuild/easyblocks/p/picard.py

```diff
@@ -72,13 +72,116 @@ def install_step(self):
 
     def sanity_check_step(self):
         """Custom sanity check for picard"""
-        jar_files = ['picard']
-        if LooseVersion(self.version) < LooseVersion('1.115'):
-            jar_files.append('sam')
+        """All versions prior to 1.124 have these jar files"""
+        if LooseVersion(self.version) < LooseVersion('1.124'):
+            jar_files = [
+                'picard-%s' % self.version,
+                
+                'AddOrReplaceReadGroups',
+                'BamIndexStats',
+                'BamToBfq',
+                'BuildBamIndex',
+                'CalculateHsMetrics',
+                'CheckIlluminaDirectory',
+                'CleanSam',
+                'CollectAlignmentSummaryMetrics',
+                'CollectGcBiasMetrics',
+                'CollectInsertSizeMetrics',
+                'CollectMultipleMetrics',
+                'CollectRnaSeqMetrics',
+                'CollectTargetedPcrMetrics',
+                'CompareSAMs',
+                'CreateSequenceDictionary',
+                'DownsampleSam',
+                'EstimateLibraryComplexity',
+                'ExtractIlluminaBarcodes',
+                'ExtractSequences',
+                'FastqToSam',
+                'FilterSamReads',
+                'FixMateInformation',
+                'IlluminaBasecallsToFastq',
+                'IlluminaBasecallsToSam',
+                'IntervalListTools',
+                'MakeSitesOnlyVcf',
+                'MarkDuplicates',
+                'MarkIlluminaAdapters',
+                'MeanQualityByCycle',
+                'MergeBamAlignment',
+                'MergeSamFiles',
+                'MergeVcfs',
+                'NormalizeFasta',
+                'QualityScoreDistribution',
+                'ReorderSam',
+                'ReplaceSamHeader',
+                'RevertSam',
+                'SamFormatConverter',
+                'SamToFastq',
+                'SortSam',
+                'SplitVcfs',
+                'ValidateSamFile',
+                'VcfFormatConverter',
+                'ViewSam',
+            ]
```

Comment:
hmm, do we want this one per line? I'd collapse this by listing multiple items on a single line, but keeping it down to max 120 chars line length?